sqlitecluster/SQLiteNode.cpp

```diff
@@ -756,7 +756,7 @@ bool SQLiteNode::update() {
 
```

Comment:
My sense:
* Debug - Information that's only important while debugging
* Info - General information about what's happening
* Hmmm - Unexpected or curious information, but not necessarily a problem
* Warn - Sign of a problem, but not necessarily unrecoverable
* Alert - A real problem that should be dealt with